src/disk_interface_test.cc

```diff
@@ -42,6 +43,15 @@ struct DiskInterfaceTest : public testing::Test {
     return fclose(f) == 0;
   }
 
+#ifdef _WIN32
+  bool Touch(const wchar_t* path) {
+    FILE* f = _wfopen(path, L"w");
+    if (!f)
```

Comment:
Proper curly braces please

Reply:
This is autoformatet by clang format.

Reply:
clang-format doesn't add `{}`, I think that's what he meant.